package.json

```
@@ -155,5 +155,8 @@
   "publishConfig": {
     "access": "public"
   },
-  "types": "./dist/index.d.ts"
+  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=14.0.0 <20.0.0"
```

```suggestion
    "node": ">=12 <20.0.0"
```
We actually support v12.x and up

---

Ahh I apologise. Reading the previous comment got me a little confused.